### ISSUE TITLE
JHipster registry error 500 in call management services

### ISSUE THREAD

**Issue by ivan86to** [jhipster/jhipster-registry#361]
##### **Overview of the issue**

NullPointerException in 
io.github.jhipster.registry.security.oauth2.OAuth2ClientCredentialsService because retrieveNewAccessToken method have call to UAA commented.

```
Caused by: java.lang.NullPointerException: null
	at org.springframework.security.jwt.JwtHelper.decode(JwtHelper.java:45)
	at io.github.jhipster.registry.security.oauth2.OAuth2ClientCredentialsService.getAccessToken(OAuth2ClientCredentialsService.java:40)
	at io.github.jhipster.registry.filters.UaaTokenRelayFilter.run(UaaTokenRelayFilter.java:51)
	at com.netflix.zuul.ZuulFilter.runFilter(ZuulFilter.java:117)
	at com.netflix.zuul.FilterProcessor.processZuulFilter(FilterProcessor.java:193)
	... 111 common frames omitted
```

##### **Reproduce the error**
Generate Gateway + UAA 
Jhipster registry in Administration -> Metrics, Logs ect.. 
Error 500 NullPointerException when call 

##### **Related issues**
#313 
##### **Suggest a Fix**
Uncomment from row 64 to 71

##### **JHipster Registry Version(s)**
5.0.1

##### **Browsers and Operating System**
- [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by gabsone** [comment]
Hi!,
some issue ... how to resolve ?

**Comment by ivan86to** [comment]
In class io.github.jhipster.registry.security.oauth2.OAuth2ClientCredentialsService uncomment row from 64 to  71 for re-enable call to uaa service

**Comment by gabsone** [comment]
Hello thanks for the help,
now no longer exceptions are thrown, but when I select metrics or swagger api from the combobox, the registry logs out

**Comment by msyzzm** [comment]
Same issue in image: jhipster/jhipster-registry:v5.0.2

**Comment by juliensadaoui** [comment]
Fixed by #392 cc @pascalgrimaud
